hack/test-templates.sh

```diff
@@ -284,6 +316,9 @@ if [[ -n ${CHECKS["port-forwards"]} ]]; then
 	if [ "${NAME}" = "opensuse" ]; then
 		limactl shell "$NAME" sudo zypper in -y netcat-openbsd
 	fi
+	if [ "${NAME}" == "wsl2" ]; then
```

Comment:
Probably this should check `limactl shell "$NAME" command -v dnf`

Reply:
Yes, this will be better. There is low probability to still face incompatibility with dnf distro, but with a different name of netcat package, but this could be fixed if it ever pops out.

Reply:
Another minor problem that it will result in a duplicate run for "fedora" case. It would result in noop install, but just is not 100% clean